docusaurus.config.js

```diff
@@ -93,7 +93,7 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       colorMode: {
-        defaultMode: 'dark',
+        defaultMode: 'light',
```

Comment:
Why changing to the light mode? Dark is more comfortable for long reads

Reply:
Really? Ok, we can leave it dark then.